administrator/components/com_templates/src/Model/TemplateModel.php

```diff
@@ -1177,9 +1177,9 @@ public function createOverride($override)
             $name         = end($explodeArray);
             $client       = ApplicationHelper::getClientInfo($template->client_id);
 
-            if (stristr($name, 'mod_') != false) {
+            if (stristr($name, 'mod_')) {
                 $htmlPath   = Path::clean($client->path . '/templates/' . $template->element . '/html/' . $name);
-            } elseif (stristr($override, 'com_') != false) {
+            } elseif (stristr($override, 'com_')) {
```

Comment:
Same as above, use strict comparison
```suggestion
            } elseif (stristr($override, 'com_') !== false) {
```